packages/ssx-sdk/src/ssx.ts

```diff
@@ -49,8 +50,19 @@ export class SSX {
     }
   }
 
-  /** Request the user to sign in, and start the session. */
-  async signIn(): Promise<SSXSession> {
+  /** 
+   * Request the user to sign in, and start the session. 
+   * @param signInOpts - Additional options to customize sign-in behavior
+   * @returns Object containing information about the session and ENS (depending on signInOpts param)
+   */
+  async signIn(
+    signInOpts: {
+      /* Enables ENS Domain resolution on client side */
+      resolveEnsDomain?: boolean,
+      /* Enables ENS Avatar resolution on client side */
+      resolveEnsAvatar?: boolean,
+    } = {}
```

Comment:
I don't like this, from an API design perspective. I think configuring this should live in the ssxConfig. Perhaps the config can have a flag for rendering on client vs server instead of adding it where you call sign in.